README.md

```diff
@@ -29,3 +29,8 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Trello config
+To run the application you will need to add a directory on your local machine called "trello_config", containing two files:
```

Comment:
I'd probably also mention the trello board setup as well (i.e. the board and list names)